mod-orgs/examples/organization_collection.sample

```diff
@@ -0,0 +1,188 @@
+      "name": "GOBI",
+      "code": "AQ-GOBI",
+      "description": "This is Yankee Book Peddler.",
+      "status": "Active",
+      "language": "en-us",
+      "isVendor": true,
+      "aliases": [
+        {
+          "value": "YBP",
+          "description": "AKA"
+        },
+        {
+          "value": "Yankee Book Peddler",
+          "description": "Formerly known as"
+        }
+      ],
+      "addresses": [
+        {
+          "address": {
+            "addressLine1": "10 Estes Street",
+            "addressLine2": "",
+            "city": "Ipswich",
+            "stateRegion": "MA",
+            "zipCode": "01938",
+            "country": "USA"
+          },
+          "categories": [
+            "0e3f9680-ab06-4565-af64-609b7364e6eb",
+            "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6",
+            "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
+            "56288fe8-8037-44da-8395-01d2d106dc54"
+          ],
+          "language": "en"
+        }
+      ],
+      "phone_numbers": [
+        {
+          "phone_number": {
+            "phone_number": "19789999999"
+          },
+          "categories": [
+            "0e3f9680-ab06-4565-af64-609b7364e6eb",
+            "996ecd31-7ca4-4d8d-9bbf-bc94dff5f6c6"
+          ],
+          "language": "en-us"
+        }
+      ],
+      "emails": [
+        {
+          "email": {
+            "value": "noreply@folio.org",
+            "description": "Main"
+          },
+          "categories": [
+            "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
+            "56288fe8-8037-44da-8395-01d2d106dc54"
+          ],
+          "language": "en-us"
+        }
+      ],
+      "urls": [
+        {
+          "url": {
+            "value": "http://folio.org",
+            "description": ""
+          },
+          "language": "en-us",
+          "categories": [
+            "112ae2e4-88ae-4fa5-a75b-2379d2035e52",
+            "56288fe8-8037-44da-8395-01d2d106dc54"
+          ],
+          "notes": "note"
+        }
+      ],
+      "contacts": [
+        "9a665b22-9fe5-4c95-b4ee-837a5433c95d"
+      ],
+      "agreements": [
+        {
+          "name": "History Follower Incentive",
+          "discount": 10,
+          "reference_url": "http://my_sample_agreement.com",
+          "notes": "note"
+        }
+      ],
+      "erp_code": "AQ-GOBI-HIST",
+      "payment_method": "EFT",
+      "access_provider": true,
+      "governmental": true,
+      "licensor": true,
+      "material_supplier": true,
+      "vendor_currencies": [
+        "USD",
+        "CAD",
+        "GBP"
+      ],
+      "claiming_interval": 30,
+      "discount_percent": 10,
+      "expected_activation_interval": 1,
+      "expected_invoice_interval": 5,
+      "renewal_activation_interval": 1,
+      "subscription_interval": 365,
+      "liable_for_vat": false,
+      "tax_id": "TX-GOBI-HIST",
+      "tax_percentage": 5,
+      "edi": {
+        "vendor_edi_code": "AQ-GOBI-HIST",
```

Comment:
can the edi have fields called vendor?

Reply:
I _think_ the EDI fields are vender-specific, and don't apply to ordinary organizations.

Reply:
If that is true please change the description of the field in organization.json file